GATEWAY/src/providers/tilesApi.service.ts

```diff
@@ -12,12 +12,25 @@ export class TilesApi {
   private virtualTiles: VirtualTile[] = [];
   private loginData: LoginData;
   private activeApp: Application;
+  flagThen: boolean = false;
+  flagCatch: boolean = false;
 
   constructor(private http: Http,
               private storage: Storage) {
   }
 
   /**
+   * The following code will mailny be used for getting private parameters
```

Comment:
I forgot to publish the comment to the review -.- I said that i dont think those methods should be there, especially the getTilesApi and setClient in mqtt and that Id rather have the code that needed them untested if there is no other way to test it.